tests/functional/scripts/rstuf-admin-ceremony.py

```diff
@@ -13,6 +13,7 @@ def _run(input, selection, input_file):
     setting_file = os.path.join(folder_name, ".rstuf.yml")
     test_settings = Dynaconf()
     test_settings.SERVER = "http://repository-service-tuf-api"
+    test_settings.HEADERS = None
```

Comment:
Why is it required to set up `HEADERS` explicitly? 

Isn't this an optional argument? 

Same for `metadata sign` and `metadata update`.